buildpack-run.sh

```diff
@@ -1,9 +1,38 @@
 #!/usr/bin/env bash
 
-git clone git@github.com:MTES-MCT/ecobalyse-private.git
+echo "-> Install 'transcrypt' to /usr/local/bin"
 
-# Scalingo should set the $SOURCE_VERSION variable
-./bin/checkout-ecobalyse-private-branch.sh $SOURCE_VERSION
+mkdir -p "$PWD/.local/bin"
+export PATH="$PWD/.local/bin":$PATH
+wget https://raw.githubusercontent.com/elasticdog/transcrypt/016b2e4b31951be5ea96233d8d2badef9c9836b6/transcrypt -O "$PWD/.local/bin/transcrypt"
+chmod +x "$PWD/.local/bin/transcrypt"
+echo "PATH: $PATH"
+
+echo "-> Cloning 'ecobalyse' public repo at $SOURCE_VERSION"
+mkdir ecobalyse
+cd ecobalyse
+# Trick to avoid cloning the whole repo
+# Instead, only fetch the current commit
+git init
+git remote add origin https://github.com/MTES-MCT/ecobalyse.git
+# depth=1 as we don't need the history
+git fetch --depth 1 origin $SOURCE_VERSION
+git checkout FETCH_HEAD
```

Comment:
Now I think about this, how about getting the raw files directly? They should be available from `https://raw.githubusercontent.com/MTES-MCT/ecobalyse/$SOURCE_VERSION/public/data/textile/processes_impacts.json` and save us a little build time here

Reply:
Meh I just reread about the missing .git env in scalingo, scratch that I guess.

Reply:
Indeed, `transcrypt` needs a "real" git repo to work properly, so I don't really have the choice here.